packages/backend/src/Application.ts

```diff
@@ -115,7 +115,7 @@ export class Application {
     )
 
     const statusController = new StatusController(
-      priceRepository,
+      priceUpdater,
```

Comment:
Why? `PriceUpdater.getStatus` just proxies the call to repository, it does not bring any additional logic.

Reply:
It is more of a contextual choice. We treat PriceUpdater as a worker, so to ask him about his state of work seems more intuitive for me. I am open to rethink it if u have other perspective on it

Reply:
The end game is to put only updaters in StatusController, so person looking at the code will see that this is a status of updaters work

Reply:
I see your point. I am OK with this, but let's stay consistent - we can add this interface for all statuses in a separate PR, for now let's keep repositories.